### DISCUSSION TITLE
Playback speed

### DISCUSSION THREAD

**Discussion by polishExperiment** [quodlibet/quodlibet#3520] [Ideas]
Wouldn't it be awesome to have playback speed UI control or plugin?

And a keyboard shortcut to speed up or slow down playback by 0.1.

**Comment by declension** [comment]
@polishExperiment have you seen the Pitch plugin (_Audio Pitch / Speed_)? It does this kind of thing

Could have shortcuts maybe, but see #1961 #2377 #2919

**Reply by polishExperiment** [reply]
![Screenshot from 2021-03-29 03-22-00](https://user-images.githubusercontent.com/584970/112775708-f69fba80-903d-11eb-80ac-85e64baeb9ad.png)

I have quod libet v4.4. This is a screenshot of my plugin list when I search for `audio` to find `Audio Pitch / Speed` plugin.
Where can I find this plugin you are speaking of and how to install and use it?

**Reply by declension** [reply]
Hmm, should be there.
![Screenshot from 2021-06-06 11-13-49](https://user-images.githubusercontent.com/3322808/120920756-4f33a980-c6b8-11eb-8b42-631553f1d61a.png)


Try clicking _show errors_. It lools like the plugin failed to initialise. 

Also if you run QL with `--debug` from the command line that can produce useful debug information.
